js/chat/chat.scss

```diff
@@ -48,6 +52,13 @@ shiny-chat-container {
       }
     }
   }
+
+  // Styling for card suggestions
+  .card[data-suggestion]:hover,
+  .card:has([data-suggestion]) {
+    color: var(--bs-link-color, #007bc2);
+    border-color: rgba(var(--bs-link-color-rgb), 0.5);
+  }
```

Comment:
I like the hover state change but I have a feeling that `:has([data-suggestion])` might be a bit too aggressive, do you have examples in mind?

Reply:
Or as a baseline, maybe just changing the border color is enough?

Reply:
Oh yea, good call, I don't think we want that selector. At first I was trying to put suggestions inside the card, but it makes sense to me not support that second case